orchagent/bufferorch.cpp

```diff
@@ -11,8 +11,6 @@ extern sai_queue_api_t *sai_queue_api;
 extern sai_switch_api_t *sai_switch_api;
 extern sai_buffer_api_t *sai_buffer_api;
 
-extern PortsOrch *gPortsOrch;
-
```

Comment:
I understand the final squash part. Small incremental changes helps reviews to understand the story author trying to tell. And this suggestion didn't mean to ask you making the change for this review. So we are good here.

I think we could also revisit the decision of squash and pull in the future.